lib/Sabberworm/CSS/OutputFormat.php

```diff
@@ -41,6 +41,10 @@ class OutputFormat {
 	// This is what’s printed after the comma of value lists
 	public $sSpaceBeforeListArgumentSeparator = '';
 	public $sSpaceAfterListArgumentSeparator = '';
+
+	// This is what’s printed after the operators in a calc function
+	public $sSpaceBeforeCalcListArgumentSeparator = ' ';
```

Comment:
Since the spaces around `-` and `+` operators are mandatory, their output should not be affected by `OutputFormat`.